SConstruct

```diff
@@ -1248,6 +1248,48 @@ for library in ( "GafferUI", ) :
 	if int( env["QT_VERSION"] ) > 4 :
 		addQtLibrary( library, "Widgets" )
 
+#########################################################################################################
+# Repair Symlinks on Windows
+#########################################################################################################
+
+# Windows does not support symlinks except with special (non-default) privileges.
+# When cloning the repository git will create symlink source files as a text
+# file with the symlink target as its content. 'fileOrigin' is a dictionary of the form
+# fileSource: fileTarget used by installers to check for overriding the file's origin.
```

Comment:
Let me see if I understand correctly, based on the little bit of googling I've just done. Windows won't _create_ symlinks without special privileges, but it will _read_ them if they have already been created. When `git clone` is run without privileges, it makes these funky text files instead of the symlinks. When we're installing files, if we see a funky text file, we manually read its contents and copy the file that it references instead. Correct? Ugh!

Rather than litter all the `InstallAs()` call sites with extra code, can we override the `INSTALL` construction variable documented on [this page](https://scons.org/doc/1.2.0/HTML/scons-user/a4774.html) instead? If that works, it would have the benefit of isolating the workarounds from the build logic itself.

Reply:
> Ugh!

Very, very ugh! But yes, you are correct. I didn't know about overriding `INSTALL`, that should make it much cleaner.

Reply:
I forgot to cherry pick that from my MSVC branch, it's in https://github.com/GafferHQ/gaffer/pull/4524/commits/9f79bc2beda4c3f4bdcc65c02d236db37e7e72c0 now.